src/limb.rs

```diff
@@ -58,21 +50,19 @@ pub(crate) type WideSignedWord = i64;
 #[cfg(target_pointer_width = "64")]
 pub type Word = u64;
 
-/// Signed integer type that corresponds to [`Word`].
-#[cfg(target_pointer_width = "64")]
-pub(crate) type SignedWord = i64;
-
 /// Wide integer type: double the width of [`Word`].
 #[cfg(target_pointer_width = "64")]
 pub type WideWord = u128;
 
-/// Signed wide integer type: double the width of [`SignedWord`].
-#[cfg(target_pointer_width = "64")]
-pub(crate) type WideSignedWord = i128;
-
 /// Highest bit in a [`Limb`].
 pub(crate) const HI_BIT: usize = Limb::BITS - 1;
 
+/// A boolean value returned by constant-time `const fn`s.
+/// `Word::MAX` signifies `true`, and `0` signifies `false`.
+// TODO: should be replaced by `subtle::Choice` or `CtOption`
+// when `subtle` starts supporting const fns.
+pub type CtChoice = Word;
```

Comment:
Seems like this could benefit from a few constants to codify the contract?

```suggestion
pub type CtChoice = Word;

pub const CT_FALSE: CtChoice = 0;
pub const CT_TRUE: CtChoice = CtChoice::MAX;
```

Reply:
Yes, I guess it's an intermediate solution between the current one and making `CtChoice` its own type.

Reply:
A newtype for `Word` might make sense, although maybe as a followup

Reply:
Added a newtype.